src/writeGraph.hpp

```diff
@@ -43,11 +44,19 @@ namespace shasta {
             const std::map<typename Graph::vertex_descriptor, VertexAttributes>&,
             const std::map<typename Graph::edge_descriptor, EdgeAttributes>&,
             ostream&);
+
+        template<class Graph> void writeOrderedSvg(
```

Comment:
Please add some comments here and/or in the `.cpp` file to describe what this does and the meaning of the arguments.